### ISSUE TITLE
Gradient update for softmax with cross entropy?

### ISSUE THREAD

**Issue by act65** [garibaldu/comp421#7]
So we have figured out the update rule for a real valued vector with the sum of the squared difference. sum((y-T)^2)
What is the update for a softmax with cross entropy?

**Comment by garibaldu** [comment]
That's what we in the trade refer to as a good question.
